backend/djangoindia/api/views/communication.py

```diff
@@ -33,6 +35,29 @@ def post(self, request, *args, **kwargs):
             )
 
 
+class UnsubscribeAPIView(generics.GenericAPIView):
+    def delete(self, request, token=None):
+        try:
+            unsubscribe_token = token
```

Comment:
@prakkhar03, we should avoid using try-except unless there’s code where we’re uncertain about how it might fail. Also you can use token directly to check condition no need to store at another variable